packages/cache/src/index.ts

```diff
@@ -0,0 +1,180 @@
+import { Redis } from 'ioredis';
+import { LRUCache } from 'lru-cache';
+import { logger } from '@prairielearn/logger';
+import * as Sentry from '@prairielearn/sentry';
+import assert from 'node:assert';
+
+class Cache {
+  enabled = false;
+  type = 'none';
+  memoryCache?: LRUCache<string, string>;
+  redisClient?: Redis;
+  keyPrefix = 'prairielearn-cache:';
```

Comment:
Nit: since `keyPrefix` is a mandatory value, we don't need a value here. This can be `''` (the empty string) to keep the types simple.

Reply:
Removed this value in [120c12b](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/120c12b2b370c18d126ea7b451843f8ac7ff2f8a).